steam/manifest.py

```diff
@@ -949,7 +949,8 @@ def __init__(
 
         self.language_support = {
             Language.from_str(language): support
-            for language, support in common.get("supported_languages", MultiDict()).items()
+            for language, support in (common.get("supported_languages") or MultiDict()).items()
+            if not isinstance(common.get("supported_languages"), str)
```

Comment:
I lied I prefer multidict here
```suggestion
            if isinstance(common.get("supported_languages"), MultiDict)
```